### DISCUSSION TITLE
请问如何配置七牛图床？

### DISCUSSION THREAD

**Discussion by dengchangdong** [Qexo/Qexo#200] [Q&A] [answered]
请问大佬使用七牛作为图床应该怎么配置了？

**Comment by am-abudu** [comment, marked as answer]
使用S3协议 参考https://developer.qiniu.com/kodo/4086/aws-s3-compatible

**Reply by dengchangdong** [reply]
> 使用S3协议 参考https://developer.qiniu.com/kodo/4086/aws-s3-compatible

已搞定，谢谢！
